src/bots/telegram/client.js

```diff
@@ -222,6 +226,17 @@ function prepareMessage(msg: Telegram$Message): string {
   return text;
 }
 
+function prepareQuote(quote) {
+  if (!quoteLength) {
+    return quote;
+  }
+
+  const prepared = quote
+    .split('\n').slice(0, 3).join('\n').slice(0, quoteLength);
+  const isMatched = prepared === quote;
+  return prepared + (isMatched ? '' : '...');
```

Comment:
there is a separate character representing ellipsis … (U+2026) instead of 3 dots

Reply:
It seems to me that showing UTF-symbols to people who are in jabber it can be dangerous.

They are not seldom under linux and some of them under terminal

Reply:
done